docs/CONTRIBUTE.md

```diff
@@ -0,0 +1,55 @@
+This repository is the deployed version of the main repository of Software Discovery Tool as a submodule and the data repository as a sub submodule.
+Any contribution to SDT should be done in the main repository.
+
+# How To Deploy
+The whole SDT deployment is maintained by:
+- [cicd.yml]() workflow
+- [config_build.py]() to create new sections in the distro config file
+- [package_build.py]() to update existing distro files to latest versions
+
+# Contribute to the deployment workflow
+
+## Contribute to deployment script
+The deployment uses [appleboy's ssh workflow](https://github.com/appleboy/ssh-action)
+The deployment is thus triggered at every new pusb into the main branch.
+The script section uses the required server credentials to be able to host the tool.
+```yml
+script: |
+          cd /opt/software-discovery-tool-deploy
+          sudo chown -R sdtuser .
+          git checkout main
+          git pull origin main --recurse-submodules
+          sudo cp -u -r software-discovery-tool/ ../software-discovery-tool/
+          sudo chown -R apache:apache /opt/software-discovery-tool
+          sudo -u apache /opt/software-discovery-tool/bin/config_build.py
+          sudo apachectl restart
+```
```

Comment:
Since this has the potential to change and we don't want to have to update in 2 places, maybe just link again to cicd.yml instead of including this text inline?